### DISCUSSION TITLE
Column Ordering on Default Grid View when using visibleGridView

### DISCUSSION THREAD

**Discussion by cancan101** [pimcore/pimcore#16898] [Q&A / Support] [answered]
Is there any way to control the column ordering for the default grid view when using `visibleGridView` to show a given column? Right now, we are using `visibleGridView` to control the columns shown in the default grid view for new users on our system. However, the column ordering is not ideal and we would like to specify a different default. Is that possible?

**Comment by markus-moser** [comment, marked as answer]
@cancan101 This is not possible just by using the `visibleGridView` checkboxes. There the order is like it is (I guess based on the order in the class definition). So if you want to change the order it's necessary to save it as grid configuration and share it with other users/roles.
